src/containers/ErrorPageHandler/index.jsx

```diff
@@ -92,7 +92,7 @@ export class ErrorPageHandler extends PureComponent {
 
     if (erroredApi.error && erroredApi.error.status === 401) {
       this._setLastUrlPath();
-      userManager.forceSignoutRedirect();
+      createUserManager().forceSignoutRedirect();
```

Comment:
I've never liked this flow of forcing a logout if we get a 401, so maybe we could revise it